app/models/patron.rb

```diff
@@ -1,4 +1,5 @@
 class Patron < ApplicationRecord
     has_many :checkouts
+    attr_accessor :email
```

Comment:
why email is `attr_accessor` not part of database?

Reply:
this i a mistake from my end, actually when i made the database of the patron, i forgot to add the column of email, that's why i use "attr_accessor" to create a virtual attribute of email for the Patron model.